packages/dashboard-frontend/src/contexts/WorkspaceActions/DeleteWarning/index.tsx

```diff
@@ -0,0 +1,119 @@
+/*
+ * Copyright (c) 2018-2024 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+
+import {
+  Button,
+  ButtonVariant,
+  Modal,
+  ModalVariant,
+  Text,
+  TextContent,
+  TextVariants,
+} from '@patternfly/react-core';
+import { ExternalLinkAltIcon } from '@patternfly/react-icons';
+import React from 'react';
+
+import { WantDelete } from '@/contexts/WorkspaceActions';
+
+export type Props = {
+  isOpen: boolean;
+  wantDelete: WantDelete;
+  onClose: () => void;
+};
+
+export class WorkspaceActionsDeleteWarning extends React.PureComponent<Props> {
```

Comment:
@olexii4 just to make sure, this warning will be shown only if there is another running workspace?

Reply:
@ibuziuk This warning will be shown only if there is user tries to delete a workspace and there is another running workspace in the namespace. We have proper tests: https://github.com/eclipse-che/che-dashboard/pull/1318/files#diff-3e8373e8b719dd1c0523b227158bb01cc7b5345bb86fd91410d77ceab1adb357R24